hardhat.config.ts

```diff
@@ -56,6 +57,11 @@ const config: HardhatUserConfig = {
     },
   },
 
+  gasReporter: {
+    enabled: true,
+    currency: "ETH"
```

Comment:
can we use USD instead?

Reply:
@joshualyguessennd  also, you can fix the missing price data by setting the token and gasPriceApi options in your config.